components/ModalDropdown.js

```diff
@@ -345,51 +371,22 @@ export default class ModalDropdown extends Component {
           highlighted && styles.highlightedRowText,
           highlighted && dropdownTextHighlightStyle,
         ]}
+        {...dropdownTextProps}
       >
         {item}
       </Text>
     ) : (
       renderRow(item, index, highlighted)
     );
-    const preservedProps = {
+
+    const touchableProps = {
       key,
       accessible,
       onPress: () => this._onRowPress(item, index, separators),
+      ...renderRowProps
     };
 
-    if (TOUCHABLE_ELEMENTS.find(name => name === row.type.displayName)) {
```

Comment:
This did not work anymore with newer React Native versions. It was always `undefined` in my tests.